ballista/ui/scheduler/.dockerignore

```diff
@@ -0,0 +1,3 @@
+node_modules
+build
+.dockerignore
```

Comment:
Do we really need to add `.dockerignore` to `.dockerignore` ?

Reply:
Not necessarily. Just avoids copying it while building the image. Adding it or not adding it doesn't make big difference. But this link https://codefresh.io/docker-tutorial/not-ignore-dockerignore-2/ suggests to not ignore it.

Reply:
I'll fix this in the upcoming commit. 👍